policy-engine/src/config.rs

```diff
@@ -1,4 +1,4 @@
-// Copyright 2018 Alex Crawford
+// Copyright 2018 - Cincinnati authors.
```

Comment:
We can probably just drop these headers. Red Hat's policy is to omit them.

Reply:
I assumed you meant the whole header (license et al.), so I completely dropped them all.